uwblueprint/llsc
# Add basic readme and pre-commit

## Notion ticket link

[Set up linting and formatting pre-commit hooks](https://www.notion.so/uwblueprintexecs/Set-up-linting-and-formatting-pre-commit-hooks-11410f3fb1dc80d6bd9ae7469cc0f283?pvs=4)



## Implementation description
* Setup basic pre-commit hooks commented out ones that are currently breaking/causing issues will look into them later/if needed
* Wrote quick startup guide in readme for backend


## Steps to test
1. run ./setup.sh and ensure pre-commit is installed properly (script can be removed and just add the pip install/brew install to read me)
2. pre-commit run or git commit -m/git push on changes
3. should lint and refactor changes if needed. (will fail if it requires any change)


## Checklist
